### DISCUSSION TITLE
[Feature request] Add more info to output before error: "There are still unapplied migrations after 60 seconds."

### DISCUSSION THREAD

**Discussion by patsevanton** [apache/airflow#33537] [Q&A]
Hello! Thanks for airflow!
Please Add more info to output before error: "There are still unapplied migrations after 60 seconds."

Code https://github.com/apache/airflow/blob/eae22cec9c87e8dad4d6e8599e45af1bdd452062/airflow/utils/db.py#L617

Feature request from issue https://github.com/apache/airflow/issues/15340

**Comment by kaxil** [comment]
Like I mentioned in https://github.com/apache/airflow/issues/15340 -- Migrations are run in a different container

https://github.com/apache/airflow/blob/54adc4463bfa2b8999ed495f3fadc8cf21900991/chart/templates/migrate-database-job.yaml#L21-L33

^^ It is run via Kubernetes `Job` and name of container is [`run-airflow-migrations`](https://github.com/apache/airflow/blob/54adc4463bfa2b8999ed495f3fadc8cf21900991/chart/templates/migrate-database-job.yaml#L62-L77).

Whereas scheduler and Webserver pods have an init-container that just wait for the migrations to be applied:

https://github.com/apache/airflow/blob/master/chart/templates/scheduler/scheduler-deployment.yaml#L97-L107
